cleanlab/dataset.py

```diff
@@ -152,11 +152,11 @@ def find_overlapping_classes(
     Parameters
     ----------
     labels : np.ndarray, optional
-      An array of shape ``(N,)`` of noisy labels, i.e. some labels may be erroneous.
-      Elements must be in the set 0, 1, ..., K-1, where K is the number of classes.
-      All the classes (0, 1, ..., and K-1) MUST be present in ``labels``, such that
+      An array of shape ``(N,)`` of noisy labels for the dataset, i.e. some labels may be erroneous.
```

Comment:
Clarify "Provides dataset-level and class-level overviews of issues in your classification dataset." for continuity?

Reply:
good call, done